[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph

anijain2305 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #152494
* #152490
* #152383
* __->__ #152384
* #152581
* #152547

For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

Before
![image](https://github.com/user-attachments/assets/4ae14530-3314-4dfa-9297-58f9e3ee4b9c)

After
![image](https://github.com/user-attachments/assets/c1457687-2396-49a7-986b-ef6145fcbf46)


https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

anijain2305 committed:
[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

Before
![image](https://github.com/user-attachments/assets/4ae14530-3314-4dfa-9297-58f9e3ee4b9c)

After
![image](https://github.com/user-attachments/assets/c1457687-2396-49a7-986b-ef6145fcbf46)


https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

Before
![image](https://github.com/user-attachments/assets/4ae14530-3314-4dfa-9297-58f9e3ee4b9c)

After
![image](https://github.com/user-attachments/assets/c1457687-2396-49a7-986b-ef6145fcbf46)


https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

malfet commented:
@pytorchbot merge revert -m "Broke CI" -c ignoredsignal

malfet commented:
@pytorchbot revert -m "Broke CI, see https://hud.pytorch.org/hud/pytorch/pytorch/52cbcac640dc1d8c12a6f8308799c75d45c6fc4a/1?per_page=50&name_filter=pull%20%2F%20linux-focal-cuda12.6-py3.10-gcc11-sm89%20%2F%20test&mergeEphemeralLF=true" -c ignoredsignal

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

Before
![image](https://github.com/user-attachments/assets/4ae14530-3314-4dfa-9297-58f9e3ee4b9c)

After
![image](https://github.com/user-attachments/assets/c1457687-2396-49a7-986b-ef6145fcbf46)


https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 committed:
Update on "[inductor][invoke_subgraph] Remove assertion checks for outputs of invoke_subgraph"


For invoke_subgraph, input assertions are good. We don't need output assertions. This is the tlparse

Before
![image](https://github.com/user-attachments/assets/4ae14530-3314-4dfa-9297-58f9e3ee4b9c)

After
![image](https://github.com/user-attachments/assets/c1457687-2396-49a7-986b-ef6145fcbf46)


https://manifold.edge.x2p.facebook.net/v0/read/tree/logs/.tmppQg3F8/index.html?bucketName=tlparse_reports&apiKey=tlparse_reports-key&withPayload=1&timeoutMsec=10000

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

anijain2305 commented:
@pytorchbot merge
